.github/dependabot.yml

```diff
@@ -11,6 +16,18 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+      - dependency-name: actions/cache
+      - dependency-name: actions/checkout
+      - dependency-name: actions/setup-python
+      # Managed by cisagov/skeleton-docker
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/github-script
+      # - dependency-name: actions/upload-artifact
+      # - dependency-name: docker/build-push-action
+      # - dependency-name: docker/login-action
+      # - dependency-name: docker/setup-buildx-action
+      # - dependency-name: docker/setup-qemu-action
```

Comment:
These lines should be uncommented.

Reply:
@jsf9k i can go ahead and uncomment them, but fyi this change was inherited from the skeleton: https://github.com/cisagov/skeleton-docker/blob/develop/.github/dependabot.yml

Reply:
@jsf9k i see that it is meant to be uncommented in downstream projects. resolved with [1d6a572](https://github.com/cisagov/domain-manager-api/pull/486/commits/1d6a572f6786d709cd55d22403ee864280f24805)